src/Hubbup.Web/DataSources/GitHubDataSource.cs

```diff
@@ -0,0 +1,295 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net.Http;
+using System.Net.Http.Headers;
+using System.Threading.Tasks;
+using Hubbup.Web.Models;
+using Hubbup.Web.Utils;
+using Microsoft.Extensions.Logging;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Serialization;
+
+namespace Hubbup.Web.DataSources
+{
+    public class GitHubDataSource : IGitHubDataSource
+    {
+        private const string GraphQlEndPoint = "https://api.github.com/graphql";
```

Comment:
Tell that to [System.Net.EndPoint](https://msdn.microsoft.com/en-us/library/system.net.endpoint(v=vs.110).aspx) ;). I'll fix it anyway though.